src/main/java/me/kavin/piped/utils/LiquibaseHelper.java

```diff
@@ -28,6 +28,8 @@ public static void init() throws Exception {
 
         // register YugabyteDB database
         DatabaseFactory.getInstance().register(new liquibase.ext.yugabytedb.database.YugabyteDBDatabase());
+        // register HsqlDatabase database
+        DatabaseFactory.getInstance().register(new liquibase.database.core.HsqlDatabase());
```

Comment:
Is this really necessary? We only add YugabyteDB since it's not included in the core liquibase.

Reply:
good catch! I will try without